nms/packages/magmalte/app/views/subscriber/SubscriberTable.js

```diff
@@ -206,69 +207,119 @@ function SubscriberActionsMenu(props: {onClose: () => void}) {
   const enqueueSnackbar = useEnqueueSnackbar();
   const ctx = useContext(SubscriberContext);
   const successCountRef = useRef(0);
+  const [
+    subscriberAction,
+    setSubscriberAction,
+  ] = useState<SubscriberActionType>('add');
   const handleClick = event => {
     setAnchorEl(event.currentTarget);
   };
 
   const handleClose = () => {
     setAnchorEl(null);
   };
+
   /**
-   * Post subscriber chunk or update subscriber(s) if param edit is set to true
+   * Delete Subscribers using a csv file or selecting subscribers with a checkbox in subscriber delete dialog.
    *
-   * @param {Array<SubscriberInfo>} subscribers Array of subscribers to Add or Edit.
-   * @param {boolean} edit Update existing subscriber(s) if set to true.
+   * @param {selectedSubscribers} selectedSubscribers Array of subscriber IMSI to delete
+   * @param {Array<SubscriberInfo>} subscribers Array of subscribers to delete
    */
-  const saveSubscribers = async (
+  const deleteSubscribers = async (
+    selectedSubscribers: Array<string>,
     subscribers: Array<SubscriberInfo>,
```

Comment:
Remove the `subscriber: Array<SubscriberInfo` argument

```suggestion
  const deleteSubscribers = async (
    selectedSubscribers: Array<string>,
```

Two reasons:
1. As a general rule, we should avoid making functions that can be called with unexpected inputs. In this case, it's possible to pass in `selectedSubscribers` and `subscribers`, and while it's benign here, it's a best practice to avoid this sort of thing.
2. The caller can convert `Array<SubscriberInfo>` into `Array<string>` trivially, and we then avoid the complexity  in this function.